setup.py

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env python
-# encoding: utf-8
 
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
 tests_require = [
     "mock",
     "pytest",
     "pytest-ruff",
     "pytest-cov",
     "pytest-timeout",
+    "ruff==0.3.1",
```

Comment:
Why are we pinning the ruff version?

Reply:
My thinking was that it would be very annoying if the linting got out of sync because the versions were different, or if the CI was using a different version than local.  Don't know if that is a reasonable fear though

Reply:
That makes sense to me. Then we can manually update the version independently of other changes we're making. Something similar happened on my most recent Jetstream PR (where linting changes ended up being bundled with functional changes and it was difficult to review).

Reply:
I worry that this will lead to mozanalysis being stuck on 0.3.1 forever. I think we should remove this pinning as part of the work done on dependency management that comes out of [this ADR](https://github.com/mozilla/mozanalysis/pull/216). That will allow it update like any other package, and we'll just have to be intentional about how we update packages to avoid scenarios like the one @danielkberry described.

Reply:
This is a really good point, I added a comment to #202 so we don't forget